Don't truncate system table column headings

mhuth opened the pull request:
@AllenBW, my attempt at https://projects.engineering.redhat.com/browse/RHCLOUD-8282.  Not sure if this is the right way to go about fixing this tho and the last_seen column treatment is a bit experimental.  I've hidden it for a bit when the screen width really squeezes the system name column.  Feel free to close this if you've got something else (ie better :smile:) in mind.

mhuth commented:
Full column names appear on all screen sizes, but the Last Seen column is visible and hidden depending on screensize.  Good or :vomiting_face: idea?

![2xl_screensize](https://user-images.githubusercontent.com/4008744/89279074-4d5d1c00-d68a-11ea-91dd-8d7f599603aa.png)

![lg_screensize](https://user-images.githubusercontent.com/4008744/89279231-78477000-d68a-11ea-9e78-2abbbef8730f.png)

![md_screensize](https://user-images.githubusercontent.com/4008744/89279254-7f6e7e00-d68a-11ea-8b1c-9dd6cc8f71b4.png)

terezanovotna commented:
Hiding the last column is not the greatest idea. 
Is there any other way we can see all the columns but also read the table titles without abbreviating them?

LI0131 commented:
@mhuth you might check out `pfReactTable.fitContent`. I used that for a similar issue on the recs table 🤷‍♂️

terezanovotna commented:
![Screen Shot 2020-08-04 at 15 22 32](https://user-images.githubusercontent.com/36040135/89298970-563bf680-d666-11ea-874b-e5680a88be05.png)
I took a screenshot of current abbreviation and there should be enough space to fit it all.

AllenBW commented:
`.fitContent` might not work here as the numbers are real small (tiny content), n the column header size is what we're lookin' to expand... 🤔

mhuth commented:
Ok @AllenBW, @LI0131, @terezanovotna, I removed the column hiding idea.  It was pretty :vomiting_face: anyway.  But I've added some CSS that also might be a bit controversial :scream:.  I've decreased the spacer between the heading text and the sorting icon (it was 1rem and now its 0.25rem), and I've set the right padding for the th and td elements to 0px.  These together save a bit more space and still look ok (I think so anyway).  I've kept `pfReactTable.wrappable` as it seems to be about the same as `fitContent`.  Anyway, LMK what you think :thinking: 

![space_saving_measures](https://user-images.githubusercontent.com/4008744/89381893-3297af80-d73d-11ea-8f43-170bbe3fd0fe.png)

AllenBW commented:
That last seen column has some odd spacing... gonna look at it 👁️ n then probably merge this 🤠

AllenBW commented:
ok commit i pushed up does this
<img width="1438" alt="Screen Shot 2020-08-05 at 8 11 30 AM" src="https://user-images.githubusercontent.com/6640236/89411421-61893380-d6f3-11ea-992e-2fdb5ba7c0d9.png">
<img width="862" alt="Screen Shot 2020-08-05 at 8 11 18 AM" src="https://user-images.githubusercontent.com/6640236/89411424-6352f700-d6f3-11ea-9621-6ca67357c3d4.png">
